__tests__/routes/recipes.test.ts

```diff
@@ -1,9 +1,7 @@
 import request from "supertest";
-import { DataSource } from "typeorm";
-
 import app from "../../src/app";
+import "reflect-metadata";
```

Comment:
I thought this import had to be done just in server.ts
Was the test not working without it?

Reply:
You were right, it is not necessary to import the reflect metadata